bug: regression after 0.23.0 to 0.24.0 update

https://github.com/google/zx/actions/runs/3058307300/jobs/4934368868#step:5:9
```
Cannot read properties of undefined (reading 'flags')
Error: Process completed with exit code 1.
```

https://github.com/SamVerschueren/tsd/compare/v0.23.0...v0.24.0#diff-77e120f3feff540d10e6e6e104aa06b7a7d08cc27d1da30666fd899a080f4736R27
```js
	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
	const maybeAlias = checker.getSymbolAtLocation(expression)!;
	const symbol = maybeAlias.flags & SymbolFlags.Alias ?
		checker.getAliasedSymbol(maybeAlias) :
		maybeAlias;
```

--- a/source/lib/parser.ts
+++ b/source/lib/parser.ts
@@ -24,21 +24,23 @@ export const extractAssertions = (program: Program): Map<Assertion, Set<CallExpr
 
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			const maybeAlias = checker.getSymbolAtLocation(expression)!;
-			const symbol = maybeAlias?.flags & SymbolFlags.Alias ?
-				checker.getAliasedSymbol(maybeAlias) :
-				maybeAlias;
+			if (maybeAlias) {
+				const symbol = maybeAlias.flags & SymbolFlags.Alias ?
+					checker.getAliasedSymbol(maybeAlias) :
+					maybeAlias;
 
-			const identifier = symbol?.getName();
+				const identifier = symbol.getName();
 
-			// Check if the call type is a valid assertion
-			if (assertionFnNames.has(identifier)) {
-				const assertion = identifier as Assertion;
+				// Check if the call type is a valid assertion
+				if (assertionFnNames.has(identifier)) {
+					const assertion = identifier as Assertion;
 
-				const nodes = assertions.get(assertion) ?? new Set<CallExpression>();
+					const nodes = assertions.get(assertion) ?? new Set<CallExpression>();
 
-				nodes.add(node);
+					nodes.add(node);
 
-				assertions.set(assertion, nodes);
+					assertions.set(assertion, nodes);
+				}
 			}
 		}
 

--- a/source/lib/parser.ts
+++ b/source/lib/parser.ts
@@ -22,8 +22,7 @@ export const extractAssertions = (program: Program): Map<Assertion, Set<CallExpr
 				node.expression.name :
 				node.expression;
 
-			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-			const maybeAlias = checker.getSymbolAtLocation(expression)!;
+			const maybeAlias = checker.getSymbolAtLocation(expression);
 			if (maybeAlias) {
 				const symbol = maybeAlias.flags & SymbolFlags.Alias ?
 					checker.getAliasedSymbol(maybeAlias) :

--- a/source/lib/parser.ts
+++ b/source/lib/parser.ts
@@ -24,11 +24,11 @@ export const extractAssertions = (program: Program): Map<Assertion, Set<CallExpr
 
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			const maybeAlias = checker.getSymbolAtLocation(expression)!;
-			const symbol = maybeAlias.flags & SymbolFlags.Alias ?
+			const symbol = maybeAlias?.flags & SymbolFlags.Alias ?
 				checker.getAliasedSymbol(maybeAlias) :
 				maybeAlias;
 
-			const identifier = symbol.getName();
+			const identifier = symbol?.getName();
 
 			// Check if the call type is a valid assertion
 			if (assertionFnNames.has(identifier)) {

